skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BtSenderActivity.java

```diff
@@ -239,14 +246,17 @@ public void onTick(long millisUntilFinished) {
 
             @Override
             public void onFinish() {
-                alertDialog.show();
+                if (!(thisActivity).isFinishing()) {
+                    alertDialog.show();
+                }
             }
         }.start();
     }
 
     @Override
     protected void onResume() {
         super.onResume();
+        senderService.startUploading(formIds, mode);
```

Comment:
This will be called whenever user goes to device home screen and opens the app. So do you think this is the right place to call this?

Reply:
Sorry, I will fix this soon 😢